dev-tools/mage/integtest.go

```diff
@@ -202,11 +197,11 @@ func NewIntegrationRunners(path string, passInEnv map[string]string) (Integratio
 		}
 		tester, ok := globalIntegrationTesters["docker"]
 		if !ok {
-			return nil, fmt.Errorf("docker integration test runner not registered")
+			return nil, fmt.Errorf("docker integration test runner not registered: %w", err)
```

Comment:
`err` is nil here, so this would be better as `errors.New("docker integration test runner not registered")` probably via a var.

Reply:
Fixed, I mixed this up removing pkg/errors from this file. I will likely do a separate PR to remove pkg/errors entirely from beats, since that can mostly be done with tooling.